tools/api-markdown-documenter/src/test/test-data/simple-suite-test/test-suite-a.api.json

```diff
@@ -1,7 +1,7 @@
 {
 	"metadata": {
 		"toolPackage": "@microsoft/api-extractor",
-		"toolVersion": "7.47.5",
+		"toolVersion": "7.52.13",
```

Comment:
Updated the version this file was generated with. Accounts for some small differences.